gui/src-tauri/tauri.conf.json

```diff
@@ -46,8 +46,8 @@
     "windows": [
       {
         "title": "Spicy Launcher",
-        "width": 955,
-        "height": 635,
+        "width": 1280,
+        "height": 720,
```

Comment:
Do we really need to increase the window size of the launcher?

Reply:
I for one would appreciate that change. It feels unnecessarily small on my 13’ MacBook monitor. The changelog text could also use more room for wider text-wrapping.